app/configurator/components/empty-right-panel.tsx

```diff
@@ -2,15 +2,12 @@ import { Trans } from "@lingui/macro";
 import { Typography } from "@mui/material";
```

Comment:
Here the whole `empty-right-panel.tsx` could be removed 😀

Reply:
Done 👍